plugins/scaffolder-backend/src/scaffolder/tasks/TaskWorker.ts

```diff
@@ -44,67 +45,116 @@ export class TaskWorker {
 
   async runOneTask(task: Task) {
     try {
-      const { actionRegistry, logger } = this.options;
+      const { actionRegistry } = this.options;
 
       const workspacePath = path.join(
         this.options.workingDirectory,
         await task.getWorkspaceName(),
       );
       await fs.ensureDir(workspacePath);
+      await task.emitLog(
+        `Starting up work with ${task.spec.steps.length} steps`,
```

Comment:
```suggestion
        `Starting up task with ${task.spec.steps.length} steps`,
```

just for verbiage alignment, i wonder if replacing work with task here makes sense?

is your view "work" was a generic term for "lots of tasks" or?

Reply:
I agree that task make more sense. I view it as one task which consists of multiple steps.